addons/account_asset/account_asset.py

```diff
@@ -400,51 +407,92 @@ def _get_move_check(self):
         self.move_check = bool(self.move_id)
 
     @api.multi
-    def create_move(self, post_move=True):
+    def create_move(self, post_move=True, group_entries=False):
         created_moves = self.env['account.move']
-        for line in self:
```

Comment:
Not sure that we will be able to remove a lot of duplicated code anyway. We will replace a dict by a function that creates a dict, but with all the values in the dict are given as parameters.